site/docs/v1/tech/tutorials/gate-accounts-until-verified.adoc

```diff
@@ -0,0 +1,131 @@
+---
+layout: doc
+title: Gate Users Until They Verify Their Email
+description: How to gate users' email accounts until verified.
+---
+
+== Overview
+
+As of version 1.27.0, FusionAuth can be configured to gate users' accounts until their email has been verified. While users successfully authenticate, they won't be able to proceed further until they have verified possession of the email address they registered with. 
+
+To enable email verification gating, do the following:
+
+* Configure the tenant with your email server
+* Configure the tenant to enable email gating
+* Optionally customize the email verification template
+* Enable self service registration
+* Configure your application with a redirect URL
+
+Let's step through each of these, but first, there are some required prerequisites.
+
+== Prerequisites
+
+This tutorial assumes you are running FusionAuth version 1.27.0 or greater. If you aren't, head on over to the link:/docs/v1/tech/installation-guide/[installation guide] or the link:/docs/v1/tech/installation-guide/upgrade/[upgrade guide] to update to a version with support for this feature.
+
+This tutorial also assumes you have a paid edition and valid license. If you need to buy an edition, please visit the link:/pricing/editions/[pricing page]. If you need to activate your instance, please visit the link:/docs/v1/tech/reactor/[Reactor page].
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+== Configure the Tenant Email Server
+
+Log in to the FusionAuth administrative user interface. You could do all of this configuration with the API or kickstart, but for this tutorial you'll use the UI.
+
+Navigate to [breadcrumb]#Tenants -> Your Tenant# and then select the [breadcrumb]#Email# tab. Scroll to the [breadcrumb]#SMTP settings# section. Configure this with your SMTP server information. If you are testing locally, you can use a mock email server like https://mailcatcher.me/[mailcatcher].
+
+image::tutorials/gate-accounts-until-verified/tenant-smtp-settings.png[Tenant SMTP settings,width=1200,role=bottom-cropped]
+
+Ensure a test email is delivered. If you have any issues, follow the steps in the link:/docs/v1/tech/troubleshooting/#troubleshooting-email[email troubleshooting documentation].
+
+Save the configuration.
+
+== Enable Email Verification Gating
+
+Navigate to [breadcrumb]#Tenants -> Your Tenant# and then select the [breadcrumb]#Email# tab. Scroll to the [breadcrumb]#Email verification settings# section. 
+
+* Enable [field]#Verify email#. 
+* Choose your email template. The default template is `Email verification`, and that will work fine for this tutorial.
+* Update the [field]#Verification strategy#. This can be either `Clickable link` or `Form field`. The former sends an email with a link to verify the user's address. The latter emails a code; the user will provide this code to FusionAuth. For this tutorial, use `Clickable link`.
```

Comment:
This seems weird to me with the semicolon. "The latter emails a code; the user will provide this code to FusionAuth." Perhaps it is grammatically fine, but figured I'd call it out.